src/ol/layers/MapboxStyleLayer.js

```diff
@@ -67,6 +68,20 @@ class MapboxStyleLayer extends Layer {
     }
   }
 
+  /**
+   * Listens to click events on the layer.
+   * @param {function} callback Callback function, called with the clicked
+   *   features (https://openlayers.org/en/latest/apidoc/module-ol_Feature.html),
+   *   the layer instance and the click coordinate.
+   */
+  onClick(callback) {
```

Comment:
I have introduced the onClick registration function for the MapboxStyleLayer, including tests.

Reply:
This function is already implemented here : https://github.com/geops/mobility-toolbox-js/blob/master/src/common/layers/Layer.js#L264